jenkins/docker/javascript.Dockerfile

```diff
@@ -10,6 +10,9 @@ RUN apt-get install -y wget gnupg \
 	&& wget -qO - https://www.mongodb.org/static/pgp/server-5.0.asc | apt-key add - \
 	&& echo "deb [ arch=amd64,arm64 ] https://repo.mongodb.org/apt/ubuntu focal/mongodb-org/5.0 multiverse" \
 	| tee /etc/apt/sources.list.d/mongodb-org-5.0.list \
+	&& wget http://archive.ubuntu.com/ubuntu/pool/main/o/openssl/libssl1.1_1.1.1f-1ubuntu2.16_amd64.deb \
```

Comment:
why 1.1 needed?

Reply:
mongodb requires openssl 1.1

Reply:
It seems in the next release of mongodb 6, the openssl 1.1 requirement might be removed.

https://jira.mongodb.org/browse/SERVER-62300